install-deps.sh

```diff
@@ -243,6 +243,30 @@ else
 		    ceph-libboost-thread1.67-dev \
 		    ceph-libboost-timer1.67-dev
                 ;;
+            *Bionic*)
+                ensure_decent_gcc_on_ubuntu 7 bionic
+                sudo add-apt-repository "deb http://security.ubuntu.com/ubuntu xenial-security main"
+                sudo apt-get update
```

Comment:
use `$SUDO` instead.

Reply:
Thanks for your review.
use "$SUDO" now.